src/App.js

```diff
@@ -1,63 +1,34 @@
-import logo from "./logo.svg";
-import "./App.css";
-
-// Import the functions you need from the SDKs you need
-import { initializeApp } from "firebase/app";
-import { getFirestore, collection, addDoc } from "firebase/firestore";
-// TODO: Add SDKs for Firebase products that you want to use
-// https://firebase.google.com/docs/web/setup#available-libraries
-
-// Your web app's Firebase configuration
-// For Firebase JS SDK v7.20.0 and later, measurementId is optional
-const firebaseConfig = {
-  apiKey: process.env.REACT_APP_API_KEY,
-  authDomain: process.env.REACT_APP_AUTH_DOMAIN,
-  projectId: process.env.REACT_APP_PROJECT_ID,
-  storageBucket: process.env.REACT_APP_STORAGE_BUCKET,
-  messagingSenderId: process.env.REACT_APP_MESSAGING_SENDERID,
-  appId: process.env.REACT_APP_APP_ID,
-  measurementId: process.env.REACT_APP_MEASUREMENT_ID,
-};
+import React from "react";
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 
-// initialization
-const app = initializeApp(firebaseConfig);
-const db = getFirestore();
-
-// do all of this in a useEffect so it only runs once
-const docRef = await addDoc(collection(db, "users"), {
-  first: "Ada",
-  last: "Lovelace",
-  born: 1815,
-});
+import "./App.css";
 
-console.log("Document writting with ID: ", docRef.id);
+import Header from './includes/components/header';
 
-const docRef2 = await addDoc(collection(db, "users"), {
-  first: "Marcin",
-  last: "Jaczynski",
-  born: 2000,
-});
+import Home from './includes/pages/Home';
+import About from './includes/pages/About';
+import PasteTOS from './includes/pages/PasteTOS';
+import Request from './includes/pages/Request';
+import NoPage from './includes/pages/NoPage';
 
-console.log("Document writting with ID: ", docRef2.id);
+import TermsPage from "./includes/pages/terms/termsPage";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Test Text Update
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <div className="main-container">
+        <Header />
+        <Routes>
+          <Route index path="/" element={<Home />} />
+          <Route path="/about" element={<About />} />
+          <Route path="/paste-tos" element={<PasteTOS />} />
+          <Route path="/request" element={<Request />} />
+          <Route path="/terms/:pageName" element={<TermsPage />} /> {/* /terms: termsId */}
```

Comment:
is termsId meant to be uncommented here? if not please remove comment

Reply:
Thanks. Resolved